.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.8"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
```

Comment:
SO nice that we built out this .txt file!

Reply:
Agreed @sienna-kopf, so friendly!